supervisor/resolution/checks/addon_pwned.py

```diff
@@ -26,6 +26,11 @@ def enabled(self) -> bool:
             return False
         return super().enabled
 
+    @enabled.setter
+    def enabled(self, value: bool) -> None:
+        """Enable or disbable check."""
+        self.sys_security.pwned = value
```

Comment:
That is wrong. You should be able to disable the check - like we do on the property without disabling the security.

Reply:
Well... that will not work...
https://github.com/home-assistant/supervisor/blob/b70ed9a60d8be8f2cb439fac4095edb605f6017a/supervisor/resolution/checks/addon_pwned.py#L22-L27